pages/resources/channel.mdx

```diff
@@ -60,24 +60,28 @@ Type `10`, `11` and `12` are only available in API v9 and above.
+| 3     | GROUP_DM                  | A private channel between multiple users                                                                                                                           |
+| 4     | GUILD_CATEGORY            | An [organizational category](https://support.discord.com/hc/en-us/articles/115001580171-Channel-Categories-101) that contains up to 50 channels                    |
+| 5     | GUILD_ANNOUNCEMENT        | Almost identical to `GUILD_TEXT`, a channel that [users can follow and crosspost into their own guild](https://support.discord.com/hc/en-us/articles/360032008192) |
+| ~~6~~ | ~~GUILD_STORE~~           | ~~A channel in which game developers can [sell their game on Discord](https://discord.com/developers/docs/game-and-server-management/special-channels)~~           |
+| ~~7~~ | ~~GUILD_LFG~~             | ~~A channel where users can match up for various games~~                                                                                                           |
+| ~~8~~ | ~~LFG_GROUP_DM~~          | ~~A private channel between multiple users for a group within an LFG channel~~                                                                                     |
+| ~~9~~ | ~~THREAD_ALPHA~~          | ~~The first iteration of the threads feature, never widely used~~                                                                                                  |
+| 10    | ANNOUNCEMENT_THREAD       | A temporary sub-channel within a `GUILD_ANNOUNCEMENT` channel                                                                                                      |
+| 11    | PUBLIC_THREAD             | a temporary sub-channel within a `GUILD_TEXT`, `GUILD_FORUM`, or `GUILD_MEDIA` channel                                                                             |
+| 12    | PRIVATE_THREAD            | a temporary sub-channel within a `GUILD_TEXT` channel that is only viewable by those invited and those with the `MANAGE_THREADS` permission                        |
+| 13    | GUILD_STAGE_VOICE         | A voice channel for [hosting events with an audience](https://support.discord.com/hc/en-us/articles/1500005513722) in a guild                                      |
+| 14    | GUILD_DIRECTORY           | The main channel in a [hub](https://support.discord.com/hc/en-us/articles/4406046651927-Discord-Student-Hubs-FAQ) containing the listed guilds                     |
+| 15    | GUILD_FORUM               | A channel that can only contain threads                                                                                                                            |
+| 16    | GUILD_MEDIA^1^            | A forum channel that is always in gallary view                                                                                                                     |
+
+
+^1^ `GUILD_MEDIA` is currently unused as a channel type. Actual media channels are determined by the `IS_MEDIA_CHANNEL` flag
```

Comment:
Was adding more space in the `Name` field necessary?

Also I don't see a point in adding a footnote just for that one field, you could instead
```
| 16    | GUILD_MEDIA               | A forum channel that is always in gallery view, although this type is currently unused since media channels are indicated by the `MEDIA_CHANNEL` flag    |
```